src/api/app/controllers/webui/request_controller.rb

```diff
@@ -41,7 +41,11 @@ def show
       @package_maintainers = target_package_maintainers
 
       # retrieve a list of all project maintainers
-      @project_maintainers = Project.find_by_name(@bs_request.target_project_name).maintainers
+      @project_maintainers = if Project.deleted?(@bs_request.target_project_name)
```

Comment:
While I agree this code isn't amazing, refactoring how maintainers of a project are fetched is clearly out of scope for this PR. It would involve changing the [HasRelationships mixin](https://github.com/openSUSE/open-build-service/blob/fe7cdbf25f2e6a73774b6d66e499af66475e69f5/src/api/app/mixins/has_relationships.rb#L77-L82), which I won't do now.